Simplenote/Classes/SPMarkdownParser.m

```diff
@@ -52,6 +52,15 @@ + (NSString *)htmlHeader
 
     NSString *css = [NSString stringWithContentsOfURL:[[NSBundle mainBundle] URLForResource:self.cssPath withExtension:nil]
                                              encoding:NSUTF8StringEncoding error:nil];
+
+    // Check if increase contrast is enabled.  If enabled amend CSS to include high contrast colors
+    //
+    if (@available(iOS 13.0, *)) {
+        if ([[UITraitCollection currentTraitCollection] accessibilityContrast] == UIAccessibilityContrastHigh) {
+            NSString *contrast = [NSString stringWithContentsOfURL:[[NSBundle mainBundle] URLForResource:self.contrastCssPath withExtension:nil]
+                                                         encoding:NSUTF8StringEncoding error:nil];
```

Comment:
Perhaps it's worth extracting this to an API? (duplicates the mechanism in L53).

```
- (NSString *)loadCSSAtPath:(NSString *)path;
```